### PR TITLE
Input mapper

### PR DESCRIPTION
**Pull request by bsutherland333**
To make ROSplane more usable, I've implemented a node that enables different forms of flight modes. This lets the user easily isolate and test any portion of the autopilot along any axis in any combination (for example, command roll angle while altitude and airspeed is managed by the path follower). It also lets the user fly with much higher level control, making the pilot's job easier (control pitch and roll angle instead of direct control surface deflection).

Since this isn't a core part of the rosplane autonomy stack, I created an additional package for anything directly relating to rosplane functionality that isn't part of the core autonomy stack. I also moved the param manager to be a library within rosplane, so other packages in the repository can make use of it.

### CODE REVIEW

File: rosplane/CMakeLists.txt [hunk 1]
```diff
@@ -33,59 +33,75 @@ find_package(Eigen3 3.3 REQUIRED NO_MODULE)
 find_package(rosflight_msgs REQUIRED)
 pkg_check_modules(YAML_CPP REQUIRED yaml-cpp)
 
-ament_export_dependencies(rclcpp rclpy)
-ament_export_include_directories(include)
```
Comment (iandareid) [inline rosplane/CMakeLists.txt:37]: Why do we not need these? I am curious, were they ever necessary? It seems like they were moved to each executable.

Comment (bsutherland333) [reply rosplane/CMakeLists.txt:37]: I don't believe so. Those two lines are used if you ever want to use ROSplane as a dependency for other packages, but you can also create and install a library like I did which is more explicit and limited in scope, I think. I'll also be the first to admit I don't understand CMake very well. At the very least, everything still works without those two lines and they weren't in the examples I looked at.

**Review by iandareid — APPROVED** [review]

File: rosplane/launch/rosplane.launch.py [hunk 2]
```diff
@@ -1,50 +1,66 @@
...
-    
+
     autopilot_params = os.path.join(
         rosplane_dir,
         'params',
         aircraft + '_autopilot_params.yaml'
     )
 
     return LaunchDescription([
+        launch.actions.DeclareLaunchArgument(
```
[the remark this answers is not in the document]
Comment (bsutherland333) [reply rosplane/launch/rosplane.launch.py:35]: I'm not certain, when the file is launched? I just know its the recommended way to pass in launch arguments at runtime with the python ros launch system.

[the remark this answers is not in the document]
Comment (bsutherland333) [reply rosplane/launch/rosplane.launch.py:35]: It is passed as an object though, it isn't directly substituted in when returned. So it happens whenever ros actually gets around to launching everything, not when the python launch file is being executed.

**Pull request merged**